src/views/about.slim

```diff
@@ -0,0 +1,27 @@
+include header
+/ About
+section.about
+  .profile
+    .profile-image
+      img alt=("Picture of Vishal") src="https://avatars0.githubusercontent.com/u/232278?v=3&s=400" /
+    .profile-info
+      h5 Vishal
+      p Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nemo dignissimos id a nisi optio quia quas quidem sequi magnam architecto eos odio iure mollitia, nulla laudantium, commodi quos alias, et.
+  .profile
+    .profile-image
+      img alt=("Picture of Ram") src="https://avatars0.githubusercontent.com/u/5818258?v=3&s=400" /
```

Comment:
You don't need the parenthesis around the alt text.  You can just have `img alt='Picture...'`